README.md

```diff
@@ -11,8 +11,24 @@ We need to test the API ecosystem behind [IO-app](https://github.com/pagopa/io-a
 ## Usage
 ```
 yarn install
+yarn generate
 yarn start 
 ```
 
+You'll be propted with a required input in which you can specify reference for the environment to test. Every input item is overridable by passing its specific env variable:
```

Comment:
may you please use this table format to document env variables:
ie. https://github.com/pagopa/io-backend#environment-variables